src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -26,6 +26,17 @@
  */
 final class ProgressBar
 {
+    public const VERBOSE = 'verbose';
+    public const VERY_VERBOSE = 'very_verbose';
+    public const DEBUG = 'debug';
+    public const NORMAL = 'normal';
+
+    private const VERBOSE_NOMAX = 'verbose_nomax';
+    private const VERY_VERBOSE_NOMAX = 'very_verbose_nomax';
+    private const DEBUG_NOMAX = 'debug_nomax';
+    private const NORMAL_NOMAX = 'normal_nomax';
+    private const NOMAX_SUFIX = '_nomax';
```

Comment:
This one should be removed, it makes no sense to me.

Reply:
I share your SMELL reaction to this. But I think the culprit is in the following lines instead:
```php
        if (!$this->max && null !== self::getFormatDefinition($format.self::NOMAX_SUFIX)) {
            $this->format = self::getFormatDefinition($format.self::NOMAX_SUFIX);
```
Assembling strings to make dynamic  code in this way if not clean. Adding a `const` to replace this magic string suffix is, indeed, doing nothing to remedy the sore eye I get either.

However fixing this was not my goal with the change. It was, instead, to remove the wide spread magic string in this script.